tests/Unit/Aspell/AspellTest.php

```diff
@@ -115,4 +115,35 @@ public function testCheckText(): void
         static::assertEquals('CCould', $issues[4]->word);
         static::assertEquals(4, $issues[4]->line);
     }
+
+    /**
+     * Test spell checking when a word contains a colon
+     *
+     * @see https://github.com/mekras/php-speller/issues/24
+     */
+    public function testCheckTestWithColon(): void
+    {
+        $source = $this->prophesize(EncodingAwareSource::class);
+        $source->getEncoding()->shouldBeCalled()->willReturn('UTF-8');
```

Comment:
You don't need to add `shouldBeCalled()` to every method your are mocking.
It is not necessary here because the only thing you want to provide is the `string` value.

What you could have also done is using a instance of `StringSource`
E.g.

```php
$source = new StringSource('S:t Petersburg är i Ryssland');
```

Because this is simply a "DataObject" which doesn't need mocking.